docs/manual/replacements_and_deletions.md

```diff
@@ -42,11 +42,45 @@ The following conditions are currently eligible for replacement:
 * `MissingPod`: This indicates a process group that doesn't have a Pod assigned.
 * `MissingPVC`: This indicates that a process group that doesn't have a PVC assigned.
 * `MissingService`: This indicates that a process group that doesn't have a Service assigned.
-* `PodPending`: This indicates that a process group where the pod is in a pending state.
+* `PodPending`: This indicates that a process group where the Pod is in a pending state.
+* `NodeTaintReplacing`: This indicates that a process group where the pod is running on a tainted Node
 
 Process groups that are set into the crash loop state with the `Buggify` setting won't be replaced by the operator.
 If the `cluster.Spec.Buggify.EmptyMonitorConf` setting is active the operator won't replace any process groups.
 
+## Automatic Replacements for Pods on Tainted Nodes
+The operator has an option to automatically replace Pods that are running on tainted Nodes. This feature is disabled by default but can be enabled by setting `automationOptions.replacements.taintReplacementOptions` and `automationOptions.replacements.taintReplacementTimeSeconds`.
+
+**Example 1.** We can use the following setup to let operator detect Pods running on Nodes with taint key `example.com/maintenance`, set the Pods' condition to `NodeTaintReplacing` if their Nodes have been tainted for 3600 seconds, and replace the Pods after 1800 seconds.
+```
+automationOptions.replacements.taintReplacementOptions = {
+    {
+        Key: example.com/maintenance
+        DurationInSeconds: 3600
+    },
+}
+automationOptions.replacements.taintReplacementTimeSeconds = 1800
+```
+
+If there are multiple Pods on tainted Nodes, the operator will simultaneously replace at most `automationOptions.replacements.maxConcurrentReplacements` Pods.
+
+**Example 2.** We can enable taint feature on all taint keys except one taint key with the following configuration:
+```
+automationOptions.replacements.taintReplacementOptions = {
+    {
+		Key:               *,
+		DurationInSeconds: 3600,
+	},
+	{
+		Key:               example.com/taint-key-to-ignore,
+		DurationInSeconds: 9223372036854775807,
+	},
+}
+```
+The operator will detect and mark all Pods on tainted Nodes with `NodeTaintDetected` condition. But the operator will not mark Pods on Nodes with taint key `example.com/taint-key-to-ignore` with `NodeTaintReplacing` condition because the key's `DurationInSeconds` is set to max of int64.
```

Comment:
`
The operator will detect and mark all Pods on tainted Nodes with `NodeTaintDetected` condition. But the operator will ignore the taint key `example.com/taint-key-to-ignore` when it adds `NodeTaintReplacing` condition to Pods, because the key's `DurationInSeconds` is set to max of int64. For example, if a Node has only the taint key `example.com/taint-key-to-ignore`, its Pods will only be marked with  `NodeTaintDetected` condition. When the Node has another taint key, say `example.com/any-other-key`, its Pods will be added `NodeTaintReplacing` condition when the other taint key has been on the Node for 3600 seconds.
`

Reply:
changed to this ^^^